webpack/sass/main.scss

```diff
@@ -0,0 +1,165 @@
+
+
+$orange1 : #EE5B32;
```

Comment:
could we name these colors more descriptively, the current variables just describe the color and the variable will have to be renamed if the colors change. it's also difficult to understand what the difference between "orange1" and "orange2" is.

see https://css-tricks.com/what-do-you-name-color-variables/ or http://geoffgraham.me/naming-sass-color-variables/ (there are a bunch of them, these are just the first two I found) for some best practices around naming colors.

Reply:
used:
brand-theme:
brand-theme-light:
brand-theme-dark:
etc.